apps/test/unit/code-studio/pd/application/TeacherApplicationTest.js

```diff
@@ -123,7 +124,7 @@ describe('TeacherApplication', () => {
         teacherApplication.findOne('FormController').props.getInitialData()
       ).to.deep.equal(parsedData);
     });
-    it('does not include saved form data if partial saving is not allowed', () => {
+    it('includes saved form data even if partial saving is not allowed', () => {
```

Comment:
This is probably not necessary, but I kept it for documentation. See full discussion here https://github.com/code-dot-org/code-dot-org/pull/43992#discussion_r770045188